05-assistive-chatbot/chatbot_api.py

```diff
@@ -47,8 +46,8 @@ def chat_engine(self):
 
 # This function cannot be async because it uses a single non-thread-safe app_state
 @app.post("/query")
-def query(message: str | Dict):
-    response = app_state.chat_engine().gen_response(message)
+def query(message: str):
```

Comment:
causes error in `populate_summaries` in ll 196, key errors for `gen_results.question`

Reply:
This fixed the underlying error? I'm a bit surprised, since I didn't think type annotations could affect runtime!